Content.Shared/Storage/EntitySystems/SharedEntityStorageSystem.cs

```diff
@@ -368,9 +368,10 @@ public bool AddToContents(EntityUid toAdd, EntityUid container, SharedEntityStor
         if (toAdd == container)
             return false;
 
-        if (TryComp<PhysicsComponent>(toAdd, out var phys))
+        if (TryComp<TransformComponent>(toAdd, out var xform))
         {
-            var aabb = _physics.GetWorldAABB(toAdd, body: phys);
+            var xformQuery = GetEntityQuery<TransformComponent>();
+            var aabb = _lookup.GetAABB(toAdd, Vector2.Zero, 0, xform, xformQuery);
```

Comment:
Should this use GetAABBNoContainer instead maybe?

Reply:
Also we get to cut out another TryComp since I no longer need that TransformComponent to call GetAABBNoContainer